go.mod

```diff
@@ -95,7 +95,6 @@ require (
 
 replace (
 	//github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
-	github.com/tektoncd/pipeline => github.com/tektoncd/pipeline v0.12.0
```

Comment:
@VaishnaviHire remember any reason we did these here?

Reply:
I think this was required for one of the dependency. But we we have a clean `go mod` , it is fine to remove this